packages/api-server/src/methods/modules/eth.ts

```diff
@@ -1209,6 +1229,70 @@ export class Eth {
 
     return [normalizedFromBlock, normalizedToBlock];
   }
+
+  // aca = auto create account
+  // Search tx by `signature_hash` when tx's `from_id = 0`
+  //
+  // `gw_get_transaction(signature_hash)`
+  //       |-> if `txWithStatus.transaction` != null
+  //             |-> found!
+  //       |-> if `txWithStatus.transaction` == null
+  //             |-> if `from_id` == null
+  //                   |-> not found!
+  //             |-> if `from_id` != null
```

Comment:
- `txWithStatus.transaction` == null AND `from_id` == null

  This indicates that an AutoCreateAccount transaction exists but has not been committed.

- `txWithStatus.transaction` == null AND `from_id` != null

  This indicates that an AutoCreateAccount transaction exists and has been committed.

Do I understand correctly?

Reply:
> txWithStatus.transaction == null AND from_id == null

This means auto create account transaction is dropped

> txWithStatus.transaction == null AND from_id != null

This means account is created but auto create account transaction is dropped

Reply:
> > txWithStatus.transaction == null AND from_id == null
> 
> This means auto create account transaction is dropped
> 
> > txWithStatus.transaction == null AND from_id != null
> 
> This means account is created but auto create account transaction is dropped

That makes a lot of sense to me.

I have another related question: is it possible that `gw_get_transaction(signature_hash) == null` AND `from_id` != null AND `gw_get_transaction(gw_tx_hash)` != null? (https://github.com/nervosnetwork/godwoken-web3/pull/445/files#diff-cce75f92de0f01f77430b0195c78d8a547ca11f2b8f818202008dce92a69ebefR1245 branch) If yes, when?

Reply:
Yes. When account is created and auto create transaction is included in mem pool.